static/app/data/forms/userFeedback.tsx

```diff
@@ -18,6 +19,23 @@ const formGroups: JsonFormObject[] = [
         help: 'Show "powered by Sentry within the feedback dialog. We appreciate you helping get the word out about Sentry! <3',
         getData: data => ({options: data}),
       },
+      {
+        name: 'sentry:feedback_user_report_notifications',
+        type: 'boolean',
+
+        label: 'Enable Crash Report Notifications',
```

Comment:
```suggestion
        label: t('Enable Crash Report Notifications'),
```

Reply:
should be translating the label & help text for "Show Sentry Branding" above too